ansible/group_vars/env_vars.tmpl

```diff
@@ -1,4 +1,5 @@
 ---
+env: ${ENVIRONMENT}
```

Comment:
Don't add the env variable as it can lead to confusion with ENV_IN_URL and ENVIRONMENT
Instead of adding it to  s3_media_repo.bucket_name, on line 49 just concatenate the name as it : 
bucket_name: ${ENVIRONMENT}-${S3_MEDIA_BUCKET_NAME}